core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-store/dot-pages.store.ts

```diff
@@ -94,12 +94,15 @@ export class DotPageStore extends ComponentStore<DotPagesState> {
                 label: this.dotMessageService.get('All'),
                 value: null
             });
-            languages.forEach((language) => {
-                languageOptions.push({
-                    label: `${language.language} (${language.countryCode})`,
-                    value: language.id
+
+            if (languages) {
```

Comment:
```suggestion
            if (!!languages.length) {
```

Reply:
![image](https://user-images.githubusercontent.com/37185433/232832063-1bc220ff-852a-4e8d-9375-94da5415ee91.png)

Reply:
so leaving it as `if (languages.length) {`